measure

```diff
@@ -9,86 +9,112 @@ import yaml
 
 from measure import Measure, ST_FAILED
 
-DESC="Prometheus measure driver for Optune by Opsani"
-VERSION="1.1.0"
-HAS_CANCEL=True
-PROGRESS_INTERVAL=30
+DESC = "Prometheus measure driver for Optune by Opsani"
+VERSION = "1.1.0"
+HAS_CANCEL = True
+PROGRESS_INTERVAL = 30
 
-CONFIG_FPATH  = os.getenv('CONFIG_FPATH', './config.yaml')
+CONFIG_FPATH = os.getenv("CONFIG_FPATH", "./config.yaml")
 
-PROM_BASE_URL = os.getenv('PROMETHEUS_ENDPOINT', 'http://prometheus:9090')
-PROM_API_PATH = '/api/v1/'
+PROM_BASE_URL = os.getenv("PROMETHEUS_ENDPOINT", "http://prometheus:9090")
+PROM_API_PATH = "/api/v1/"
 
-class Prom(Measure):
 
+class Prom(Measure):
     @staticmethod
     def get_config(path=CONFIG_FPATH):
         # read and parse config
+
+        # Load configuration from "prom" section of the config file, unless
+        #   PROMETHEUS_USE_DRIVER_NAME environment variable is defined. Use config
+        #   defined in section matching driver name in that case.
+        driver_key = "prom"
+        if os.environ.get('PROMETHEUS_USE_DRIVER_NAME', False):
+            driver_key = os.path.basename(__file__)
         try:
-            config = yaml.safe_load(open(path))['prom']
-            assert config and config.get('metrics'), 'No metrics has been provided at path "prom.metrics" ' \
-                                                     'in file located at {}'.format(path)
+            config = yaml.safe_load(open(path))[driver_key]
+            assert config and config.get("metrics"), 'No metrics has been provided at path "{}.metrics" in file located at {}"'.format(driver_key, path)
```

Comment:
FYI, we can now leverage format `f'{config.get('metric')} is not found'`

Reply:
Right, but this comes from original driver - I'm not modifying that part.